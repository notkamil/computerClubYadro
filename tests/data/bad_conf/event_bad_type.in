3
08:00 20:00
10
09:00 5 client1