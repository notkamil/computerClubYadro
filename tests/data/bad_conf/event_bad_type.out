09:00 5 client1